stdlib/public/core/CMakeLists.txt

```diff
@@ -121,7 +120,6 @@ set(SWIFTLIB_SOURCES
   ### PLEASE KEEP THIS LIST IN ALPHABETICAL ORDER ###
   Availability.swift
   Bit.swift
-  CollectionMirrors.swift.gyb
   CollectionOfOne.swift
   ExistentialCollection.swift.gyb
   Mirror.swift
```

Comment:
Hi Dave,

`_legacyMirror` is tightly integrated with the `Mirror` initializers that generate mirrors from `_reflect()` if no custom mirror is available. I don't think it can be removed until Phase 3 of [SR-88](https://bugs.swift.org/browse/SR-88) is complete, at which point we'd rip all use of `_reflect()` out of `Mirror` and use the new runtime entry points.

What do you think?